### PR TITLE
Fix/fbar 330 tabindex search

### PR DESCRIPTION
**Pull request by Camwyn**
See https://github.com/the-events-calendar/tribe-common/pull/2230

### 🎫 Ticket

[FBAR-330]

### 🗒️ Description

Making select tab friendly.

### 🎥 Artifacts 

https://www.loom.com/share/243adc4b980d429da9eed454e88543bb?sid=3050fb61-e2a5-4ac0-8581-cfef87ce97fe
### ✔️ Checklist
- [x] Ran `npm run changelog` to add changelog file(s). More info [here](https://docs.theeventscalendar.com/developer/git/changelogs/#process)
- [ ] Code is covered by **NEW** `wpunit` or `integration` tests.
- [ ] Code is covered by **EXISTING** `wpunit` or `integration` tests.
- [x] Are all the **required** tests passing?
- [x] Automated code review comments are addressed.
- [x] Have you added Artifacts?
- [x] Check the base branch for your PR.
- [x] Add your PR to the project board for the release.

[FBAR-330]: https://stellarwp.atlassian.net/browse/FBAR-330?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: vendor/tribe-selectWoo/dist/js/selectWoo.full.js [hunk 1]
```diff
@@ -988,10 +988,16 @@ S2.define('select2/results',[
...
+	var $option = $(option);
 
-    if (data.children) {
-      var $option = $(option);
+    $option.on( 'blur', function() {
+	  	$option.attr( 'data-selected', 'false' );
+    });
+	$option.on( 'focus', function() {
+		$option.attr( 'data-selected', 'true' );
+	});
```
Comment (Camwyn) [inline vendor/tribe-selectWoo/dist/js/selectWoo.full.js:998]: Something odd with the indentation here?

File: vendor/tribe-selectWoo/dist/js/selectWoo.full.js [hunk 2]
```diff
@@ -958,7 +958,7 @@ S2.define('select2/results',[
     var attrs = {
       'role': 'option',
       'data-selected': 'false',
-      'tabindex': -1
+	  'tabindex':'0',
```
Comment (Camwyn) [inline vendor/tribe-selectWoo/dist/js/selectWoo.full.js:961]: weird indentation?

File: vendor/tribe-selectWoo/dist/js/selectWoo.full.js [hunk 3]
```diff
@@ -4035,21 +4040,24 @@ S2.define('select2/dropdown/search',[
...
+          return;
+      }
       self.handleSearch(evt);
     });
 
     container.on('open', function () {
       self.$search.attr('tabindex', 0);
       self.$search.attr('aria-owns', resultsId);
-      self.$search.trigger( 'focus' );
+ //     self.$search.trigger( 'focus' );
```
Comment (Camwyn) [inline vendor/tribe-selectWoo/dist/js/selectWoo.full.js:4053]: commented out code?

**Review by stratease — APPROVED** [review]
Nice job!

**Pull request merged**